mne/decoding/receptive_field.py

```diff
@@ -183,10 +177,12 @@ def fit(self, X, y):
 
         if isinstance(self.estimator, numbers.Real):
             if self.fit_intercept is None:
-                self.fit_intercept = True
+                self.fit_intercept_ = True
+            else:
+                self.fit_intercept_ = self.fit_intercept
```

Comment:
I'd do this as `self.fit_intercept_ = True if self.fit_intercept is None else self.fit_intercept`